pkg/awsManager/vpcVpnManager.go

```diff
@@ -508,60 +477,49 @@ func DeleteVpcEndpoint(client clientpkg.Client, vpcId *string) error {
 
 	vpcNotDeleted, err := client.DeleteVpcEndpoints(&ec2.DeleteVpcEndpointsInput{VpcEndpointIds: vpcEndpointToBeDeleted})
 	if err != nil {
-		fmt.Println("Following VPC could not be deleted ", vpcNotDeleted.String())
-		fmt.Println(err)
+		logger.Error(err, "Failed to delete VPC", vpcNotDeleted.String())
 
 	} else {
-		fmt.Println("ALL VPC have been deleted successfully")
+		logger.Info("ALL VPCs have been deleted successfully")
 
 	}
 	return nil
 }
 
-func DeleteVpnConnections(client clientpkg.Client) error {
+func DeleteVpnConnections(client clientpkg.Client, logger logr.Logger) error {
 
 	// does not require pagination
 	vpnConnectionList, err := client.DescribeVpnConnections(&ec2.DescribeVpnConnectionsInput{})
 	if err != nil {
-		fmt.Println(err)
-		fmt.Println("ERROR: Unable to retreive VPN connection list ")
+		logger.Error(err, "Failed to retrieve VPN connection list")
 		return err
 	}
 
 	for _, vpnConnection := range vpnConnectionList.VpnConnections {
-
 		_, err = client.DeleteVpnConnection(&ec2.DeleteVpnConnectionInput{VpnConnectionId: vpnConnection.VpnConnectionId})
 		if err != nil {
-			fmt.Println("ERROR : cant delete VPN connection ", *vpnConnection.VpnConnectionId)
-			fmt.Println(err)
-
+			logger.Error(err, "Failed to delete VPN connection", *vpnConnection.VpnConnectionId)
 		}
-
 	}
 
 	return nil
 
 }
 
-func DetachVpnGateway(client clientpkg.Client, vpcId *string) error {
+func detachVpnGateway(client clientpkg.Client, vpcId *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file